cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -29,12 +29,14 @@ pub struct ParsedQObject {
     pub rust_struct: Option<ItemStruct>,
     /// This is a representation of the Signals enum that defines the Q_SIGNALS for the QObject
     pub signals: Option<ParsedSignalsEnum>,
+    /// This is the list of invokables that need to be implemented on the C++ object in Rust
+    ///
+    /// These will also be exposed as Q_INVOKABLE on the C++ object
+    pub invokables: Vec<ImplItemMethod>,
     /// This is the list of methods that need to be implemented on the C++ object in Rust
```

Comment:
```suggestion
    /// The list of methods that need to be implemented on the C++ object in Rust
```

`This is` starts documenting every item in this struct but it adds nothing

Reply:
Changed the wording for all of them.